qt/python/mantidqtinterfaces/mantidqtinterfaces/PyChop/PyChopGui.py

```diff
@@ -214,6 +214,11 @@ def setFreq(self, freqtext=None, **kwargs):
         # Checks for independent phases
         phases = []
         for key, widget in self.widgets.items():
+            if "MERLIN" in str(self.engine.instname) and key.endswith("Phase") and widget["Label"].isHidden():
+                # Special case for MERLIN
+                # sets the default phase for Chopper0Phase if not in "Instrument Scientist Mode"
+                phase = float(widget["Edit"].text())
```

Comment:
So I think that with this code if a user goes into instrument scientist mode, changes the value of `Chopper0Phase` in the GUI, then leaves instrument scientist mode, that the previously set value will be used rather than the default. Given that this could happen, I think that maybe the expression should be the same as in line 229 ( `phase = float(widget["Edit"].text())) % (1e6 / self.engine.moderator.source_rep)` )?

Also, I think that this is the desired behaviour of the program if the user does this, but maybe it needs to be mentioned in the [docs](https://github.com/mantidproject/mantid/blob/main/docs/source/interfaces/direct/PyChop.rst?plain=1#L65-L68)? (Something like `If this mode is then deselected, the previously inputted time delay will be used for subsequent calculations instead of the default value.`?).